pkg/gardenlet/controller/managedseed/managedseed_valueshelper_test.go

```diff
@@ -343,14 +352,112 @@ var _ = Describe("ValuesHelper", func() {
 		It("should compute the correct gardenlet chart values with bootstrap", func() {
 			result, err := vh.GetGardenletChartValues(mergedDeployment, mergedGardenletConfig(true), "bootstrap kubeconfig")
 			Expect(err).ToNot(HaveOccurred())
-			Expect(result).To(Equal(gardenletChartValues(true, "bootstrap kubeconfig")))
+			Expect(result).To(Equal(gardenletChartValues(true, "bootstrap kubeconfig", "node", 1, nil)))
 		})
 
 		It("should compute the correct gardenlet chart values without bootstrap", func() {
 			result, err := vh.GetGardenletChartValues(mergedDeployment, mergedGardenletConfig(false), "")
 			Expect(err).ToNot(HaveOccurred())
-			Expect(result).To(Equal(gardenletChartValues(false, "")))
+			Expect(result).To(Equal(gardenletChartValues(false, "", "node", 1, nil)))
+		})
+
+		Context("when HA is configured for seed ", func() {
```

Comment:
```suggestion
		Context("when HA is configured for seed", func() {
```

Reply:
Changed in 682377197fb9ada9212326ef3ebb666106b7f81a.